site/content/en/docs/tasks/run_flux_minicluster.md

```diff
@@ -0,0 +1,67 @@
+---
+title: "Run A Flux MiniCluster"
+date: 2022-02-14
+weight: 6
+description: >
+  Run a Kueue scheduled Flux MiniCluster.
+---
+
+This page shows how to leverage Kueue's scheduling and resource management capabilities when running [Flux Operator's](https://flux-framework.org/flux-operator/) MiniClusters.
+
+This guide is for [batch users](/docs/tasks#batch-user) that have a basic understanding of Kueue. For more information, see [Kueue's overview](/docs/overview).
+
+## Before you begin
+
+Check [Run A Job](/docs/tasks/run_jobs) for details on the initial cluster setup and [queue configuration](/docs/tasks/administer_cluster_quotas).
```

Comment:
```suggestion
Check [administer cluster quotas](/docs/tasks/administer_cluster_quotas) for details on the initial cluster setup.
```

Reply:
Also add a link to the guide to install flux-operator

Reply:
done